tests/system_checks/tests.py

```diff
@@ -1,19 +1,50 @@
 from django.core import checks
 from django.db import models
-from django.test import SimpleTestCase
+from django.test import TestCase, skipIfDBFeature, skipUnlessDBFeature
 from django.test.utils import (
     isolate_apps,
     override_system_checks,
 )
 
 from django_mongodb_backend.checks import check_vector_search_indexes
 from django_mongodb_backend.fields import ArrayField
-from django_mongodb_backend.indexes import VectorSearchIndex
+from django_mongodb_backend.indexes import SearchIndex, VectorSearchIndex
 
 
+@skipIfDBFeature("supports_search_indexes")
 @isolate_apps("system_checks", attr_name="apps")
 @override_system_checks([check_vector_search_indexes])
-class InvalidSearchIndexesTest(SimpleTestCase):
+class InvalidSearchIndexesTest(TestCase):
+    def test_search_requires_search_index_support(self):
+        class Article(models.Model):
+            title = models.CharField(max_length=10)
+
+            class Meta:
+                indexes = [
+                    SearchIndex(fields=["title"]),
```

Comment:
Could you please also add a VectorSearchIndex to verify that `super().check()` is called and that it works properly.